House-of-glass/src/components/Cards/Horizontal/HorizontalCard.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Ratings from '../../Ratings/Ratings'
+
+export default function HorizontalCard({CardTitle,ItemPrice,ItemDiscount,DeliveryType,ItemImage,ItemRating}) {
+  return (
+      <div className="card card--horizontal box-shadow">
+          <div className="card__contents">
+            <div className="card__content--column card__text">
+              <h5 className="header-5">{CardTitle}</h5>
+              <div className="container--row">
+                <p className="subtitle-1 item--sale text--primary">{ItemPrice}</p>
+                <p className="body-2 item--items">{ItemDiscount}</p>
+                <p className="subtitle-1 item--info text--grey">{DeliveryType}</p>
+                <p />e.   
+              </div>
+            </div>
+            <div className="card__content--row card__footer">
+              <div className="container--row">
+                <div className="btn btn--small flex-mid-center btn--primary--outline">
+                  <p className="subtitle-1 btn--text text">-</p>
+                </div>
+                <div className="btn btn--small flex-mid-center btn--primary">
+                  <p className="subtitle-1 btn--text text">1</p>
+                </div>
+                <div className="btn btn--small flex-mid-center btn--primary--outline">
+                  <p className="subtitle-1 btn--text text">+</p>
+                </div>
+                <div className="btn btn--small btn--outline--like">
+                  <span className="material-icons icon--">bookmark</span>
+                  <p className="subtitle-1 btn--txt">Save</p>
```

Comment:
I assume these are wishlist kinda functionality. Then it  makes sense to me to get the entire object and destructure it as you might have to send a post request from here.

Reply:
Yes, that does make more sense. Have called the item and destructed it now. 
Thank you for this suggestion. 
You saved me some time 😬